test/hotspot/jtreg/runtime/ClassFile/JsrRewriting.java

```diff
@@ -34,50 +34,44 @@
  * @bug 7149464
  * @requires vm.flagless
  * @library /test/lib
+ * @library /testlibrary/asm
```

Comment:
This file should change the copyright to look like:

 * Copyright (c) 2011, 2025, Oracle and/or its affiliates. All rights reserved.